tests/virtualization/universal/patch_and_reboot.pm

```diff
@@ -40,12 +40,15 @@ sub run {
     add_test_repositories;
     fully_patch_system;
 
+    # useful for debugging
```

Comment:
You probably know this does not belong here 😄

Reply:
I see what you mean :slightly_smiling_face: This commit doesn't belong there.

Reply:
Please fix it and close this thread then ;)